### ISSUE TITLE
Request a scope review (Buyers)

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#134]
**Navigation:** (update)

- Buyers
- - Research the OASIS+ contract features
- - Complete market research

**Buyers' landing:** (update)
- Remove "coming soon" from Get a scope review (content coming soon)
- Link the title to the page when it's created

**Prohibitions page**
- Link to scope review page from ""OCOs seeking further guidance on these flexibilities are encouraged to submit a scope review outlining the desired flexibility."

- [copy doc](https://docs.google.com/document/d/12IEDMPsGL2VsXTYTMnbSH2iEXN5am7XvbVpN3OigXsA/edit#heading=h.iygv7gu3pcbe) 

**Status**

- [x]  PSHC created content
- [x]  SME reviewed content
- [x]  PSHC incorporate SME feedback
- [x]  SD reviewed content
- [x]  PSHC incorporate SD feedback
- [ ]  Content is done
- [ ]  Google doc is locked
- [ ]  Design created or template identified
- [ ]  Ready for development

**Comment by klohman** [comment]
Update 5/22/24

- Still waiting on form to be live, but we can publish the page with some interim text (@rosamundtgov to update copy doc). 
- Moved this into sprint 6 backlog, but it is a nice-to-have released this sprint only if we have capacity and time. T
- This page does not need to be included in the printable Buyers' ordering guide

**Comment by rosamundtgov** [comment]
@PowderD Updated this page. Let us know what you think; if you approve of the changes, can you comment here?

**Comment by PowderD** [comment]
@rosamundtgov good to move forward

**Comment by weiwang-gsa** [comment]
@rosamundtgov just double check with you the path to this page is "buyers-guide/get-scope-review" not "buyers-guide/request-scope-review"?

**Comment by weiwang-gsa** [comment]
@rosamundtgov For this "Link to scope review page from ""OCOs seeking further guidance on these flexibilities are encouraged to submit a scope review outlining the desired flexibility."  should link text be "submit a scope review" or just "scope review"?

**Comment by weiwang-gsa** [comment]
@rosamundtgov it is ready for review at feature branch, please double check if my changes are ok in prohibitions page and buyer's guide landing page: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/wwang/request-scope-review/oasis-plus/buyers-guide/request-scope-review/

**Comment by rosamundtgov** [comment]
> https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/wwang/request-scope-review/oasis-plus/buyers-guide/request-scope-review/

Can you change "are encouraged to" to "should request a"?

**Comment by weiwang-gsa** [comment]
The menu order based on below: 
Buyer’s guide Landing page 0 
* Research the OASIS+ contract features 1
* Request a scope review 2
* Complete market research  3
* Research agencies and sellers (TBD)  4
* Get a Delegation of Procurement Authority 5
* Register in Symphony 6
* Develop the solicitation 7
* Award the task order 8
* Manage the task order 9

**Comment by weiwang-gsa** [comment]
@rosamundtgov correction is made, also I set the menu order on previous ticket, please let me know if my assumption is wrong, it is ready for review: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/oasis-plus/buyers-guide/request-scope-review/

**Comment by PowderD** [comment]
@weiwang-gsa @rosamundtgov Should this section be a link? 

we can provide Market Research as a Service (MRaS) [complete-market-research/#]mras upon request.

**Comment by weiwang-gsa** [comment]
@PowderD great catch, will make correction.

**Comment by weiwang-gsa** [comment]
@rosamundtgov per Jon's comment, would you mind verifying below section links (two links) are correct or not as I can't clearly tell from the doc where link start and ends, so I just did based on my guess.  It can be reviewed at feature branch: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/wwang/request-scope-review/oasis-plus/buyers-guide/request-scope-review/

![Image](https://github.com/GSA/oasis-plus/assets/119954777/a76e413a-b598-40d2-8639-a77fdd1c322b)

Thanks!

**Comment by rosamundtgov** [comment]
It looks correct to me -

**Comment by weiwang-gsa** [comment]
@PowderD or @ekidenda it is ready for QA review here: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/oasis-plus/buyers-guide/request-scope-review/

**Comment by ekidenda** [comment]
@weiwang-gsa both links took me to the intended destination - to fill a scope review form or complete market research. Moving to done.
